connectors/src/connectors/slack/lib/utils.ts

```diff
@@ -51,10 +51,81 @@ export const timeAgoFrom = (millisSinceEpoch: number) => {
   return seconds + "s";
 };
 
-export function internalIdFromSlackChannelId(channel: string) {
+export type SlackChannelInternalId = string;
+export type SlackThreadInternalId = string;
+export type SlackNonThreadedMessagesInternalId = string;
+
+export function isSlackChannelInternalId(
```

Comment:
nit: would use regexes here

Reply:
IMHO it would make the thing harder to reason about